Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,53 @@
+.row {
+    display: flex;
+    flex-flow: row nowrap; 
+    height: 100%;
+    margin: 2em;
+}
+
+.col {
+    border-radius: 5px;
+    margin: 1em;
+    padding: 1em;
+    position: relative;
+    width:25%;
+    
+    &:before {
+        content: "";
+        background-color: $green;
+        border-radius: 5px;
+        bottom: -15px;
+        box-shadow: -11px -11px white, -12px -12px $green, -12px -10px $green, -10px -12px $green;
+        left: -15px;
+        padding: 2em;
+        position: absolute;
+        right: -15px;
+        top: -15px;
+        z-index: -1;
+    }
+    
+    & > img {
+        height: $tech-logo-size;
+        width: 100%;
+    }
+
+    & > h3 {
+        display: block;
+        font-size: $text-big;
+        margin: 10px 0;
+        text-align: center;
+    }
+    
+    & > span {
+        display: block;
+        font-weight: bold;
+        font-size: $text-medium;
+        text-align: center;
+    }
+    
+    & > p{
```

Comment:
don't use `& > ` try to keep it less specific, also use classes instead of html tags